### PR TITLE
(Fix) O3-3554:Fix refreshing issue on packaging units and saving error on stock rules.

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements
- [x] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [x]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary
- Fixes the error saving stock rule.
- Fixes the error of not refreshing after creating at package unit.

## Screenshots
Screenshot 2024-07-08 at 14 22 22

Screenshot 2024-07-08 at 17 00 26 (2)

## Related Issue
- [O3-3554](https://openmrs.atlassian.net/browse/O3-3554)
- [O3-3312](https://openmrs.atlassian.net/browse/O3-3312)

[O3-3554]: https://openmrs.atlassian.net/browse/O3-3554?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[O3-3312]: https://openmrs.atlassian.net/browse/O3-3312?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx [hunk 1]
```diff
@@ -20,10 +22,11 @@ import { zodResolver } from "@hookform/resolvers/zod";
 import { PackageUnitFormData, packageUnitSchema } from "./validationSchema";
 import { StockItemPackagingUOMDTO } from "../../../core/api/types/stockItem/StockItemPackagingUOM";
 import { createStockItemPackagingUnit } from "../../stock-items.resource";
-import { showSnackbar } from "@openmrs/esm-framework";
-import { useTranslation } from "react-i18next";
-import styles from "./packaging-units.scss";
 import DeleteModalButton from "./packaging-units-delete-modal-button.component";
+import { handleMutate } from "../../../utils";
+import { toQueryParams, ResourceRepresentation } from "../../../core/api/api";
```
Comment (jabahum) [inline src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx:27]: i think this import is unused

File: src/stock-items/add-stock-item/stock-item-rules/add-stock-rules.component.tsx [hunk 2]
```diff
@@ -67,7 +67,10 @@ const StockRulesAddOrUpdate: React.FC<AddStockRuleProps> = ({
     isLoading: loadingRoles,
   } = useRoles({ v: ResourceRepresentation.Default });
 
-  const [formModel, setFormModel] = useState<StockRule>();
+  const [formModel, setFormModel] = useState<StockRule>({
+    enabled: true,
```
Comment (jabahum) [inline src/stock-items/add-stock-item/stock-item-rules/add-stock-rules.component.tsx:71]: does this enable it by default even at create of a rule

File: src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx (hunk 1 above)
Comment (hadijahkyampeire) [reply src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx:27]: No they are used on line 91

Comment (jabahum) [reply src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx:27]: noted

**Review by jabahum — COMMENTED** [review]
Thanks @hadijahkyampeire

**Review by jabahum — APPROVED** [review]
Thanks

**Pull request merged**